doc/sphinx/source/dbk.rst

```diff
@@ -34,3 +34,63 @@ buffer are as follows: point (0,0) is the top left of the image and each pixel c
 padding. So to get a pixel value from a certain row, add an offset of row_index * width * 3 to the index. The fourcc
 code that comes closest to this is the BI_RGB 24bpp format. Check the `include/CL/cl_exp_defined_builtin_kernels.h` for
 info on DBK attributes and `tests/runtime/test_dbk_jpeg.c` for an example program making use of the JPEG DBKs.
+
+
+exp_onnx_inference
+__________________
+
+This experimental extension proposes adding a builtin kernel that ingests an
```

Comment:
The idea of the DBK extension is that it gets updated with new versions over time, without needing to add a new extension for each DBK. Let's see how well that works and not add a new extension but just a new DBK to the DBK extension. Thus the title here should be ONNX Inference DBK or such.